Add accent color to WC Blocks price slider

Aljullu opened the pull request:
This PR adds styles to the price slider of the Filter Products by Price block. It requires the changes in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3300, which decrease the selector specificity of the default styles.

### Screenshots
#### Customizer
![Peek 2020-10-20 18-25](https://user-images.githubusercontent.com/3616980/96615478-99bb0e00-1301-11eb-90ad-bcf7af565730.gif)

#### Child theme
(Screenshots with [Bistro](https://woocommerce.com/products/bistro/))

_Before:_
<img src="https://user-images.githubusercontent.com/3616980/96570001-2053f900-12ca-11eb-8a75-8a54f243bda3.png" alt="Price filter screenshot" width="286" />

_After:_
<img src="https://user-images.githubusercontent.com/3616980/96569858-f0a4f100-12c9-11eb-8011-05227bb60277.png" alt="Price filter screenshot with custom styles" width="286" />

### How to test the changes in this Pull Request:

1. Create a page with the price filter and the All Products block.
2. In the customizer, change the accent color to something different from purple.
3. Verify the price slider has the new accent color.
4. Test a Storefront child theme too to ensure the accent color of the child theme is honored as well.

### Changelog

> Enhancement – Added styles to the Filter Products by Price block, so the accent color is used in the slider.

Aljullu committed:
Add accent color to WC Blocks price slider

Aljullu committed:
Add IE11 styles

senadir reviewed (approved):
Those changes are not being honoured on the editor, this can be addressed in a follow-up.

On initially loading this PR, I see this on frontend
<img width="1091" alt="Screen Shot 2020-11-02 at 1 14 13 PM" src="https://user-images.githubusercontent.com/6165348/97867741-b34f5300-1d0e-11eb-8ba8-c4be06992cb3.png">
In the editor, I see this:
<img width="1090" alt="Screen Shot 2020-11-02 at 1 15 35 PM" src="https://user-images.githubusercontent.com/6165348/97867752-b9ddca80-1d0e-11eb-92fc-f162775211c4.png">

When changing a color, the result is reflected on the frontend, but not the editor.

On trunk, I can see the new purple on both editor and frontend, so I don't know why is the frontend using old purple in this PR.

senadir commented:
I missed this step

> In WC Blocks directory check out the branch update/styles-source: git fetch && git checkout update/styles-source.

senadir commented:
After doing that step, I'm still seeing this issue in the editor (color not being respected).

Aljullu committed:
Fix WC Blocks styles not visible in the editor

Aljullu commented:
> Those changes are not being honoured on the editor, this can be addressed in a follow-up.

Good catch @senadir! I fixed this in 07f58462ec54ccf9d6c5ff00060f611655d9de68. The issue was affecting other blocks like Cart and Checkout (they were not honoring the button color in the editor). I moved the CSS of all blocks so it should be fixed in all.

> After doing that step, I'm still seeing this issue in the editor (color not being respected).

Ah, sorry, I just updated the PR to remove that step. It's no longer needed since that PR was merged into WC Blocks.

Aljullu commented:
I will proceed merging this since it has already been approved and this way it gets in `trunk` for tomorrow's RC. We will still have one week of testing just in case something unexpected happens.
